cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -50,9 +50,10 @@ type upgradeFunc func(u *upgradeParams) (*apis.CStorPool, error)
 
 var (
 	upgradeMap = map[string]upgradeFunc{
-		"1.0.0-1.3.0": nothing,
-		"1.1.0-1.3.0": nothing,
-		"1.2.0-1.3.0": nothing,
+		"1.0.0": nothing,
```

Comment:
try removing 'nothing' kind of entries.. after version validation, if the map doesn't have some version, it can be assumed as 'nothing'

Reply:
Done